cmd/wharf/root.go

```diff
@@ -67,6 +78,17 @@ func loadKubeconfig(overrides clientcmd.ConfigOverrides) (*rest.Config, string,
 }
 
 func execute(version app.Version) {
+	var err error
+	if rootConfig, err = config.LoadConfig(); err != nil {
+		initLoggingIfNeeded()
+		log.Error().Messagef("Config load: %s", err)
+		os.Exit(exitCodeLoadConfigError)
+	}
+
+	for _, f := range runAfterConfig {
+		f()
+	}
```

Comment:
nit: A comment here would be good, because this is a pretty ugly (but effective) solution, so a motivation is needed

```suggestion
	// Some code we want to run AFTER all init()'s and AFTER wharf-cmd-config.yml
	// has been loaded; but BEFORE cobra starts parsing all arguments as flags.
	for _, f := range runAfterConfig {
		f()
	}
```

Idk, maybe it's self-explanatory as-is. Your call

Reply:
I think a comment is warranted as well 👍🏻